OCP-4.X/roles/post-install/tasks/main.yml

```
@@ -379,3 +379,22 @@
   when: dittybopper_enable
 
 
+- name: Deploy promtail
+  block:
+    - name: Template Promtail Values
+      template:
+        src: promtail-values.yml.j2
+        dest: "{{ ansible_user_dir }}/{{ dynamic_deploy_path }}/promtail.yaml"
+    - name: Install Promtail
+      shell: |
+        helm repo add grafana https://grafana.github.io/helm-charts 
+        helm repo update
+        oc create namespace loki || true
+        oc adm policy add-scc-to-group privileged system:authenticated
+        helm upgrade --install promtail --namespace=loki grafana/promtail -f {{ ansible_user_dir }}/{{ dynamic_deploy_path }}/promtail.yaml
+      ignore_errors: yes
```

Not a huge fan of ignoring errors, it could lead to us thinking loki is working out of the box, but in reality it isn't... Hard to catch later too...